skatepark/child-theme.json

```diff
@@ -228,6 +229,11 @@
 					"letterSpacing": "0.1em"
 				}
 			},
+			"core/separator": {
+				"border": {
+					"width": "0 0 3px 0"
+				}
+			},
```

Comment:
on a first pass, I was using columns and a separator but then changed to use the caption of the image. I'm leaving this here even though it's unrelated, because it may be needed elsewhere